### PR TITLE
Add mappings for Monarch

### PR DESCRIPTION
**Pull request by kshefchek**
Reconciling biolink-model spec with our [scigraph configuration](https://github.com/monarch-initiative/scigraph-docker/blob/master/conf/monarch-load-global.yaml) of categories and node properties, as well as prefixes in our [curie map](https://github.com/monarch-initiative/dipper/blob/master/dipper/curie_map.yaml).

- SO and SEPIO additions.
- Note that IAO:0000013 (journal article) is a nibling of IAO:0000311 (publication) in IAO, we use the latter so we'll need this mapping, or we can create a subclass journal article but I imagine we don't need this resolution.
- Added foaf:Person for case, is this right?
- New Genotypic entity class, see https://github.com/biolink/biolink-model/issues/383
- Many new node properties

Closes https://github.com/biolink/biolink-model/issues/62

### CODE REVIEW

File: biolink-model.yaml [hunk 1]
```diff
@@ -2561,6 +2619,7 @@ classes:
       - NCBITaxon
     mappings:
       - WIKIDATA:Q16521
+      - NCBITaxon:1
```
Comment (kshefchek) [inline biolink-model.yaml:2622]: Tag https://github.com/biolink/biolink-model/issues/171

File: biolink-model.yaml [hunk 2]
```diff
@@ -2417,15 +2454,32 @@ classes:
...
-      with respect to severity, laterality, age of onset, and other aspects
+      with respect to severity, laterality, and other aspects
     subclass_of: "HP:0012823"
     is_a: attribute
 
+  clinical course:
+    description: >-
+      The course a disease typically takes from its onset, progression in time, and eventual resolution or death of
+      the affected individual
+    subclass_of: "HP:0031797"
```
Comment (kshefchek) [inline biolink-model.yaml:2465]: this should probably be a mapping instead of subclass_of?

File: biolink-model.yaml [hunk 3]
```diff
@@ -3045,6 +3116,7 @@ classes:
       A subcellular location, cell type or gross anatomical part
     mappings:
       - SIO:010046
+      - UBERON:0001062
```
Comment (kshefchek) [inline biolink-model.yaml:3119]: tag https://github.com/biolink/biolink-model/issues/171

File: biolink-model.yaml [hunk 4]
```diff
@@ -4211,6 +4327,32 @@ classes:
           - value: MONDO:0016419
             description: hereditary breast cancer
 
+
+  genotype to disease association:
+    is_a: association
+    comments:
+      - TODO decide no how to model pathogenicity
```
Comment (deepakunni3) [inline biolink-model.yaml:4334]: @kshefchek Is this TODO still relevant?

Comment (kshefchek) [reply biolink-model.yaml:4334]: I think this just from copying over from variant to disease association, so the TODO might still stand

Comment (deepakunni3) [reply biolink-model.yaml:4334]: 👍

**Pull request merged**